R/Chromatograms.R

```diff
@@ -98,23 +101,33 @@ setClass("Chromatograms",
          slots = c(
              backend = "ChromBackend",
              processingQueue = "list",
-             # processingQueueVariables = "character", # wait do we keep this ???
```

Comment:
we can add them later if we need them, but I think with your new approach we don't need it.